### ISSUE TITLE
F5 VM is not accessible after deploy

### ISSUE THREAD

**Issue by auto2mate** [F5Networks/terraform-gcp-bigip-module#43]
I am able to deploy the F5 VM in GCP (I cloned the repo) on existing VPC (modified the main.tf to use existing VPC/Subnets) but the VM is not responding to ping/SSH/HTTPS so can't login to the VM.

**Comment by pgouband** [comment]
Hi @auto2mate,

Is it the same issue as the one you opened in 2022?

**Comment by auto2mate** [comment]
yes, I have been struggling to access the F5 VM after deployment. I did allow the required internet access from the VM and I could see as per our firewall logs that its attempting to go over the internet on port 443 using external interface but ultimately times out. 

These are the sample IP's its trying to connect - 13.249.98.74, 13.249.98.48, 13.249.98.24......

**Comment by pgouband** [comment]
Hi @auto2mate,

I think the best is to open a ticket via https://my.f5.com/ to get the appropriate help.

**Comment by auto2mate** [comment]
After going in round and round with F5 support they are asking to collaborate and open ticket on Github itself. I am still stuck and can't deploy multi NIC F5 appliance on GCP

**Comment by janibashamd** [comment]
@RavinderReddyF5 any help here !
